src/routes/transactions/mappers/multisig-transactions/multisig-transaction.mapper.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@nestjs/common';
+import { MultisigTransaction } from '../../../../domain/safe/entities/multisig-transaction.entity';
+import { Safe } from '../../../../domain/safe/entities/safe.entity';
+import { Transaction } from '../../entities/transaction.entity';
+import { MultisigTransactionExecutionInfoMapper } from './multisig-transaction-execution-info.mapper';
+import { MultisigTransactionInfoMapper } from './transaction-info/multisig-transaction-info.mapper';
+import { MultisigTransactionStatusMapper } from './multisig-transaction-status.mapper';
+
+@Injectable()
+export class MultisigTransactionMapper {
+  constructor(
+    private readonly statusMapper: MultisigTransactionStatusMapper,
+    private readonly transactionInfoMapper: MultisigTransactionInfoMapper,
+    private readonly executionInfoMapper: MultisigTransactionExecutionInfoMapper,
+  ) {}
+
+  async mapTransaction(
+    chainId: string,
+    transaction: MultisigTransaction,
+    safe: Safe,
+  ): Promise<Transaction> {
+    const txStatus = this.statusMapper.mapTransactionStatus(transaction, safe);
+    const txInfo = await this.transactionInfoMapper.mapTransactionInfo(
+      chainId,
+      transaction,
+      safe,
+    );
+    const executionInfo = await this.executionInfoMapper.mapExecutionInfo(
```

Comment:
`await` shouldn't be required here.

Reply:
Changed in [d985d6c](https://github.com/5afe/safe-client-gateway-nest/pull/197/commits/d985d6cb5043fb4c54881c2cdab35c7007f20306)